test/smart-contract-client.test.js

```diff
@@ -0,0 +1,160 @@
+import { assertEquals, assertRejects, assert, assertStringIncludes } from 'zinnia:assert'
+import { getMinerPeerIdFromSmartContract } from '../lib/smart-contract-client.js'
+import { getMinerPeerId } from '../lib/miner-info.js'
+import { test } from 'zinnia:test'
+
+const mockPeerIdResponse = {
```

Comment:
The name `mockPeerIdResponse` is very similar to `mockEmptyPeerIdResponse`, I had troubles spotting the difference when reading the tests below. Let's use more distinct names please.

For example:

```suggestion
const mockValidPeerIdResponse = {
```

Also, the prefix "mock" is not suitable here - mocks are objects/functions with active behaviour. See https://martinfowler.com/bliki/TestDouble.html

Can we simply drop the `mock` prefix and use the names `validPeerIdResponse` and `emptyPeerIdResponse` instead?

Reply:
Addressed in 9a8a972